python/thunder/imgprocessing/transformation.py

```diff
@@ -78,3 +241,98 @@ def apply(self, im):
 
     def __repr__(self):
         return "PlanarDisplacement(delta=%s)" % repr(self.delta)
+
+
+class GridTransformer(object):
```

Comment:
Maybe move this to `utils`? Or you want it here because it's so closely related to defining transformations?